From 6b50cb9ed039fd43a35078837bd8ee19d16299fc Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Thu, 17 Oct 2019 17:17:33 +0200
Subject: [PATCH 0133/1239] arm64: mvebu: config: a8k-mcbin: enable SAR

Enabling SAR configuration option will allow to make
use of the newly added sample-at-reset infrastructure
for the A8K SoC family.

Change-Id: Ia93297654a0ebfaef2d5b73eace761914f456923
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
---
 configs/mvebu_mcbin-88f8040_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index 2cbd356109..2cfe53e80e 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -7,6 +7,7 @@ CONFIG_TARGET_MVEBU_ARMADA_8K=y
 CONFIG_NR_DRAM_BANKS=2
 CONFIG_DEBUG_UART_BASE=0xf0512000
 CONFIG_DEBUG_UART_CLOCK=200000000
+CONFIG_MVEBU_SAR=y
 CONFIG_DEBUG_UART=y
 CONFIG_AHCI=y
 CONFIG_DISTRO_DEFAULTS=y
-- 
2.29.0

